packages/opentelemetry-basic-tracer/src/Span.ts

```diff
@@ -47,24 +42,23 @@ export class Span implements types.Span {
   constructor(
     parentTracer: types.Tracer,
     spanName: string,
-    options: types.SpanOptions
+    traceId: string,
+    kind: SpanKind,
+    parentSpanId?: string,
+    traceState?: TraceState,
+    startTime?: number
   ) {
     this._tracer = parentTracer;
     this._name = spanName;
-    this._spanContext.spanId = randomSpanId();
-    this._spanContext.traceOptions = TraceOptions.SAMPLED;
-    const parentSpanContext = this._getParentSpanContext(options.parent);
-    if (parentSpanContext && isValid(parentSpanContext)) {
-      // New child span.
-      this._spanContext.traceId = parentSpanContext.traceId;
-      this._spanContext.traceState = parentSpanContext.traceState;
-      this._parentId = parentSpanContext.spanId;
-    } else {
-      // This is a root span so no remote or local parent.
-      this._spanContext.traceId = randomTraceId();
-    }
-    this._kind = options.kind || types.SpanKind.INTERNAL;
-    this._startTime = options.startTime || performance.now();
+    this._spanContext = {
+      traceId,
+      spanId: randomSpanId(),
```

Comment:
I realize we need to wait for the spec, but here's to wishing there were a way to optionally specify this line! (Needed for browser initial load tracing)

Reply:
What is the behavior you would like to see? (Just confused to what this comment is referring to in the code.)

Reply:
In reference to this comment: https://github.com/open-telemetry/rfcs/pull/8#discussion_r307942349